components/script/dom/node.rs

```diff
@@ -1788,6 +1788,11 @@ impl Node {
         copy
     }
 
+    /// https://html.spec.whatwg.org/multipage/#child-text-content
+    pub fn child_text_content(&self) -> DOMString {
+        Node::collect_text_contents(self.children())
```

Comment:
It's used [here](https://github.com/servo/servo/blob/87f7b29d65d8a0ad137e38412aef50734b24ea6c/components/script/dom/node.rs#L1952).